ClinGen/clincoded
# #1794 Display functional data

**Steps to test #1794**

1. Add Variant by CA ID
2. In the evidence view, click the **Experimental** tab
3. In the **Structured Narrative of Functional Impact** table, expect to see functional data with each PubMed article separated in tabs if there are multiple articles
4. Check links and text formatting
